fbw-a32nx/src/wasm/systems/a320_systems/src/pneumatic/wing_anti_ice.rs

```diff
@@ -486,13 +486,15 @@ impl WingAntiIceSystem {
 
         // Check if HIGH or LOW pressure
         if self.wai_consumer_pressure()
+            // FIXME use ADR static pressure
```

Comment:
Are you sure this is meant to be ADR pressure? The WAI system may use its own pressure transducer here.

Reply:
That's possible, but in that case it should be reading the static pressure from the sim.